spec/services/person_update_notifier_spec.rb

```diff
@@ -55,6 +55,7 @@ def check_send_update_reminder(expected)
       context "when no last reminder sent" do
         before { person.update(last_reminder_email_at: nil) }
 
+        # rubocop:disable RSpec/NoExpectationExample
         it "returns true when last update more than 6 months ago" do
           person.update!(updated_at: more_than_six_months_ago)
           check_send_update_reminder true
```

Comment:
It might be better to remove this method and add the expectation back to the tests as it's only 1 line. Then you don't need to disable the cop.